core/app/modules/kit-library/assets/js/components/item-header.js

```diff
@@ -159,9 +173,11 @@ export default function ItemHeader( props ) {
 				/>
 			}
 			<Header
-				startColumn={ <HeaderBackButton /> }
+				startColumn={ <HeaderBackButton kitName={ props.model.title } pageId={ props.pageId } /> }
 				centerColumn={ props.centerColumn }
 				buttons={ buttons }
+				kitName={ props.model.title }
+				pageId={ props.pageId }
```

Comment:
Why double props with the startColumn? Can it be merged?

Reply:
Merged